drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -473,12 +473,11 @@ static bool nrf5_tx_at(struct net_pkt *pkt, uint8_t *payload, bool cca)
 		.cca = cca,
 		.channel = nrf_802154_channel_get(),
 	};
-	uint32_t tx_at = net_pkt_txtime(pkt) / NSEC_PER_USEC;
+	uint64_t tx_at = net_pkt_txtime(pkt) / NSEC_PER_USEC;
 	bool ret;
 
 	ret = nrf_802154_transmit_raw_at(payload,
-					 tx_at - TXTIME_OFFSET_US,
```

Comment:
Should be safe to remove the `TXTIME_OFFSET_US` definition above as well.

Reply:
Removed the definition.